mm2src/coins/lightning.rs

```diff
@@ -174,27 +121,39 @@ impl LightningCoin {
             })
     }
 
-    fn pay_invoice(&self, invoice: Invoice) -> SendPaymentResult<(PaymentHash, PaymentInfo)> {
+    fn pay_invoice(&self, invoice: Invoice) -> SendPaymentResult<PaymentInfo> {
         self.invoice_payer
             .pay_invoice(&invoice)
             .map_to_mm(|e| SendPaymentError::PaymentError(format!("{:?}", e)))?;
         let payment_hash = PaymentHash((*invoice.payment_hash()).into_inner());
+        let payment_type = PaymentType::OutboundPayment {
+            destination: invoice.payee_pub_key().cloned(),
+        };
+        let description = Some(match invoice.description() {
```

Comment:
At least here, the description is always `Some(...)` can we use an empty string instead if there's no description?